### PR TITLE
Fixed Issue 2868, libvirt resize notify failure

### PR DESCRIPTION
**Pull request by kiwiflyer**
## Description
Fixed for https://github.com/apache/cloudstack/issues/2868
Incorrect diskpath information was being sent to virsh blockresize, so the block device size was never refreshed to reflect the new disk size.

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

## GitHub Issue/PRs

Fixes: #2868 

## Screenshots (if appropriate):
This was tested on Centos 7 with Ceph backend storage. Virtio-blk driver was used in the VM. Ubuntu 16.04 and Centos 7 VMs were tested.

Disk was 106GB:
before_106gb
Resize was used via the ACS GUI to resize the block device on ceph to 150GB.
The VM picks up the change immediately:
after_150gb

## How Has This Been Tested?
Yes, see above.

## Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/apache/cloudstack/blob/master/CONTRIBUTING.md) document.
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
Testing
- [ ] I have added tests to cover my changes.
- [ ] All relevant new and existing integration tests have passed.
- [ ] A full integration testsuite with all test that can run on my environment has passed.

### CODE REVIEW

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan test

File: scripts/storage/qcow2/resizevolume.sh [hunk 1]
```diff
@@ -92,7 +92,8 @@ notifyqemu() {
     if `virsh domstate $vmname >/dev/null 2>&1`
     then
       sizeinkb=$(($newsize/1024))
-      virsh blockresize --domain $vmname --path $path --size $sizeinkb >/dev/null 2>&1
+      devicepath=$(virsh domblklist $vmname|grep $path|tr -s ' ' | cut -d' ' -f1)
```
[the remark this answers is not in the document]
Comment (kiwiflyer) [reply scripts/storage/qcow2/resizevolume.sh]: I can change this to use awk. This is  definitely broken on Centos 7.x and Ubuntu 16.04 (tested with Ceph RBD on both). It fails silently and just produces a failure log on the host, but success is returned to the management server, so most folks probably don't realize it's broken.

[the remark this answers is not in the document]
Comment (rhtyd) [reply scripts/storage/qcow2/resizevolume.sh]: I tested with nfs based and local storage as well.

**Comment by rhtyd** [comment]
Test LGTM, will merge this.

**Pull request merged**